client/src/test/java/com/sinch/sdk/models/ConfigurationBuilderTest.java

```diff
@@ -23,6 +26,12 @@ void build() {
             .setNumbersContext(NumbersContext.builder().setNumbersUrl(NUMBERS_SERVER).build())
             .setSmsContext(
                 SmsContext.builder().setSmsRegion(SMS_REGION).setSmsUrl(SMS_SERVER).build())
+            .setConversationContext(
+                ConversationContext.builder()
+                    .setRegion(CONVERSATION_REGION)
```

Comment:
As the region is an optional parameter, do you think it's worth to add a test to verify the default value when not specified ?

Reply:
This test already exist (but not in the scope of the Builder: default value is managed by context itself):
- context is having a default value if not set: https://github.com/sinch/sinch-sdk-java/blob/feat/conversation-rebase-main/client/src/main/com/sinch/sdk/models/SmsContext.java#L13
- test against default value: https://github.com/sinch/sinch-sdk-java/blob/feat/conversation-rebase-main/client/src/test/java/com/sinch/sdk/models/SmsContextTest.java#L14